packages/muelu/src/Utils/ForwardDeclaration/MueLu_AMGXOperator_fwd.hpp

```diff
@@ -47,7 +47,7 @@
 #define MUELU_AMGXOPERATOR_FWD_HPP
 
 #include "MueLu_ConfigDefs.hpp"
-#if defined(HAVE_MUELU_EXPERIMENTAL) && defined(HAVE_MUELU_AMGX)
```

Comment:
@csiefer2 had the same change in his PR.  I guess this is script generated, but indicates that someone modified the file by hand at some point?

Reply:
Yup, looks that way. I ran the script to generate fwd declarations.